### ISSUE TITLE
Cannot install swift-sh on Arch Linux

### ISSUE THREAD

**Issue by segabor** [mxcl/swift-sh#5]
Brew install fails with error `missingLinuxMain`. Here's the console log

```
[segabor@csihuhu domain11]$ brew install mxcl/made/swift-sh
==> Tapping mxcl/made
Cloning into '/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/mxcl/homebrew-made'...
remote: Enumerating objects: 3, done.
remote: Counting objects: 100% (3/3), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 3 (delta 0), reused 2 (delta 0), pack-reused 0
Unpacking objects: 100% (3/3), done.
Tapped 1 formula (27 files, 21.9KB).
==> Installing swift-sh from mxcl/made
==> Downloading https://github.com/mxcl/swift-sh/archive/1.1.3.zip
==> Downloading from https://codeload.github.com/mxcl/swift-sh/zip/1.1.3
#=#=-#  #                                                                     
==> swift build --configuration release -Xswiftc -static-stdlib --disable-sandbox
Last 15 lines from /home/segabor/.cache/Homebrew/Logs/swift-sh/01.swift:
2019-01-14 14:04:48 +0100

swift
build
--configuration
release
-Xswiftc
-static-stdlib
--disable-sandbox

error: missingLinuxMain

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/mxcl/homebrew-made/issues
```

**Comment by mxcl** [comment]
Seems like a SwiftPM. bug where it requires a linux-test-main even though we aren't building the tests.

I can fix, but I can only test in CI.

**Comment by segabor** [comment]
Let me take a closer look.

**Comment by mxcl** [comment]
Well I'm on it, but you're welcome. The strategy I was going to use was taking the tests out of the Linux version with `#if os(Linux)`, since maintaining a Linux main is a PITA, I'm waiting for Apple to fix that themselves.

**Comment by segabor** [comment]
Tests are now covered but the project itself does not build due to Foundation differences on Linux

```
[segabor@csihuhu swift-sh]$ swift build
Compile Swift Module 'Shwifty' (6 sources)
/home/segabor/Workspace/swift-sh/Sources/Script.swift:78:17: error: value of type 'Process' has no member 'run'
            try task.run()
                ^~~~ ~~~
/home/segabor/Workspace/swift-sh/Sources/Path.swift:318:21: error: type 'NSData.WritingOptions' has no member 'atomicWrite'
            opts = .atomicWrite
                    ^~~~~~~~~~~
error: terminated(1): /usr/bin/swift-build-tool -f /home/segabor/Workspace/swift-sh/.build/debug.yaml main output:
```

**Comment by mxcl** [comment]
After #8 I'll tag a new version and bump the formula. Thanks for the report and help!

**Comment by segabor** [comment]
Just one more thing. I made a typo in my PR and also testTarget was not enabled on Linux in Package.swift. New pull request was opened for this.

Now project builds on Linux but tests break with the followin
```
[segabor@csihuhu swift-sh]$ swift build && swift test
Test Suite 'All tests' started at 2019-01-14 16:46:47.175
Test Suite 'debug.xctest' started at 2019-01-14 16:46:47.175
Test Suite 'IntegrationTests' started at 2019-01-14 16:46:47.175
Test Case 'IntegrationTests.testConventional' started at 2019-01-14 16:46:47.176
Fatal error: init(for:) is not yet implemented: file Foundation/Bundle.swift, line 57
Exited with signal code 4
```

**Comment by segabor** [comment]
I'm going to open a separate ticket for Bundle specific issue.

**Comment by mxcl** [comment]
Fixed by #8, new release tagged and can be installed with any of the support methods.
